src/rust/ide/view/graph-editor/src/component/node/output/port.rs

```diff
@@ -34,8 +35,11 @@ const PORT_OPACITY_NOT_HOVERED : f32 = 0.25;
 const SEGMENT_GAP_WIDTH        : f32 = 2.0;
 const HOVER_AREA_PADDING       : f32 = 20.0;
 const INFINITE                 : f32 = 99999.0;
+const FULL_TYPE_ONSET_DELAY_MS : f32 = 2000.0;
 
 const TOOLTIP_LOCATION : Placement = Placement::Bottom;
+const SHOW_TOOLTIP     : bool      = false;
+const SHOW_TYPE_LABEL  : bool      = true;
```

Comment:
Please add comments here describing what these constants are used for (explaining that we have left 2 implementations for now).